spec/Common.hs

```diff
@@ -55,12 +56,15 @@ enc val = TL.toStrict . TL.decodeUtf8 . encode $ val
 
 article1 :: Value
 article1 = object [ "id" .= (1 :: Int)
-                  , "date" .= ("2014-10-20T07:00:00" :: Text)
-                  , "title" .= object ["rendered" .= ("<i>Foo</i> bar" :: Text)]
-                  , "excerpt" .= object ["rendered" .= ("summary" :: Text)]
-                  , "departments" .= [ object [ "name" .= ("some department" :: Text)]]
-                  , "department" .= ("15" :: Text)
-                  ]
+                  , "date" .= ("2014-10-20T07:00:00" :: T.Text)
+                  , "modified" .= ("2014-10-20T07:00:00" :: T.Text)
+                  , "guid" .= object ["rendered" .= ("http://localhost/2014/10/a-fourth-post/" :: T.Text)]
+                  , "title" .= object ["rendered" .= ("<i>Foo</i> bar" :: T.Text)]
+                  , "excerpt" .= object ["rendered" .= ("summary" :: T.Text)]
+                  , "departments" .= [ object [ "name" .= ("some department" :: T.Text)]]
+                  , "department" .= ("15" :: T.Text)
+                  , "authors" .= [ object ["name" .= ("Emma Goldman" :: T.Text)]]
+                  , "format" .= ("standard" :: T.Text) ]
```

Comment:
Added fields expected by the feed: "modified", "guid", "authors" -- latter two are temporary I think. The entry id should be generated to point to the Offset server, not the WordPress server (although it doesn't matter if it's a real, clickable URL). The "authors" field is there because Jacobin needs multiple authors, but we need to support *at least* both guest-authors and the default user authors.  (The "format" field is a mistake.)